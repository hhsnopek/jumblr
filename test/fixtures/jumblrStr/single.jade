include ../../../lib/jumblr

+jumblrStr('hasPage', 'foo')
a(class=jumblrStr id="bar")
