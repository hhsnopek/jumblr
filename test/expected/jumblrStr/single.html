<a id="bar" class="{block:hasPage}foo{/block:hasPage}"></a>
